_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __ } from 'i18n-calypso';
+
+class ProductExpiration extends React.PureComponent {
+	static propTypes = {
+		expiryDate: PropTypes.string.isRequired,
```

Comment:
I'm not sure why the expiry date would be required, but the purchase date won't be. I think we should follow the same pattern for all the dates here.

Reply:
Fixed in 84727755f71988299f54bf4e06f3dcc99d6171ab